site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1687,7 +1688,7 @@ components:
       filterJson:
         name: filterJson
         in: query
-        description: |
+        description: |        
```

Comment:
can you please remove the trailing whitespaces?

Reply:
Done